src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -199,30 +199,59 @@ internal static bool ShowCrashErrorReportWindow(DynamoViewModel viewModel, Crash
                 {
                     string logFile = Path.Combine(cerDir.FullName, "DynamoLog.log");
 
-                    File.Copy(model.Logger.LogPath, logFile);
-                    // might be usefull to dump all loaded Packages into
-                    // the log at this point.
-                    filesToSend.Add(logFile);
+                    try
+                    {
+                        File.Copy(model.Logger.LogPath, logFile);
+                        // might be useful to dump all loaded Packages into
+                        // the log at this point.
+                        filesToSend.Add(logFile);
```

Comment:
Why do you think these lines of code need to be within a try-catch? This is not yet interacting with CER at this point.